src/reanimated2/NativeReanimated/NativeReanimated.ts

```diff
@@ -1,31 +1,34 @@
+import { NativeModules } from 'react-native';
 import { SharedValue } from '../commonTypes';
 import { Descriptor } from '../hook/commonTypes';
 
-const InnerNativeModule = global.__reanimatedModuleProxy;
-
 export class NativeReanimated {
   native: boolean;
-  useOnlyV1: boolean;
+  private InnerNativeModule: any;
 
   constructor(native = true) {
+    if (global.__reanimatedModuleProxy === undefined) {
+      const { ReanimatedModule } = NativeModules;
+      ReanimatedModule?.installTurboModule();
+    }
+    this.InnerNativeModule = global.__reanimatedModuleProxy;
     this.native = native;
```

Comment:
is this being used somewhere? Quick lookup seems to show that we never pass `false` for `native parameter, can you maybe mark that as something we want to remove in the future?

Reply:
Check `JSReanimated` c-tor, there's `super(false);` in there. Right now, `.native` is used in two places:
1. `NativeReanimatedModule.installCoreFunctions()` call
2. `requestFrame()` function